include/net/download_client.h

```diff
@@ -57,26 +63,30 @@ enum download_client_evt_id {
 	 * - EAFNOSUPPORT: Unsupported address family (IPv4/IPv6)
 	 * - EHOSTUNREACH: Failed to resolve the target address
 	 *
-	 * In case of errors on the socket during send() or recv() (ECONNRESET),
-	 * returning zero from the callback will let the library attempt
-	 * to reconnect to the server and download the last fragment again.
-	 * Otherwise, the application may return any non-zero value
-	 * to stop the download. On any other error code than ECONNRESET, the client
-	 * will not attempt to reconnect and ignores the return value.
+	 * In case of @c ECONNRESET errors, returning zero from the callback will let the
+	 * library attempt to reconnect to the server and download the last fragment again.
+	 * Otherwise, the application may return any non-zero value to stop the download.
+	 * On any other error code than ECONNRESET, the client will not attempt to reconnect
+	 * and ignores the return value.
+	 *
+	 * In case the download is stopped or completed, and the @c keep_connection flag in
+	 * @c download_client_host_cfg.download_client_host_cfg is set, the client will stay
```

Comment:
You'll need to tune the line breaks:
```suggestion
	 * In case the download is stopped or completed, and the
	 * @c download_client_host_cfg.keep_connection flag is set, the client will stay
```